modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/hemophage/hemophage_status_effects.dm

```diff
@@ -1,11 +1,11 @@
 /// How much brute damage their body regenerates per second while using blood regeneration.
-#define BLOOD_REGEN_BRUTE_AMOUNT 0.75
+#define BLOOD_REGEN_BRUTE_AMOUNT 2
 /// How much burn damage their body regenerates per second while using blood regeneration.
-#define BLOOD_REGEN_BURN_AMOUNT 0.75
+#define BLOOD_REGEN_BURN_AMOUNT 2
 /// How much toxin damage their body regenerates per second while using blood regeneration.
-#define BLOOD_REGEN_TOXIN_AMOUNT 0.5
+#define BLOOD_REGEN_TOXIN_AMOUNT 1.75
```

Comment:
Just so it's not as efficient and still makes a difference, now it's be 75% as efficient for toxins, as opposed to 87.5%.
```suggestion
#define BLOOD_REGEN_TOXIN_AMOUNT 1.5
```